lib/LedgerSMB/Scripts/payment.pm

```diff
@@ -252,14 +278,16 @@ sub pre_bulk_post_report {
     }];
     delete $request->{$_}
        for qw(action dbh);
-    return $template->render({
-        form => $request,
-        hiddens => $request,
-        columns => $cols,
-        heading => $heading,
-        rows    => $rows,
-        buttons => $buttons,
-    });
+    $request->{hiddens} = { %$request }; # prevent circular reference
+    delete $request->{form_id};
+    my $template = LedgerSMB::Template::UI->new_UI;
+    return $template->render($request, 'Reports/display_report',
```

Comment:
render has 2 parameters.

Reply:
Hmm. I thought it had 4: https://github.com/ledgersmb/LedgerSMB/blob/master/lib/LedgerSMB/Template/UI.pm#L168 ?